src/core/libmaven/mzAligner.cpp

```diff
@@ -483,7 +484,7 @@ void Aligner::alignSampleRts(mzSample* sample, vector<float> &mzPoints,ObiWarp&
     }
 }
 
-void Aligner::alignWithObiWarp(vector<mzSample*> samples,  ObiParams* obiParams,int referenceSampleIndex){
+void Aligner::alignWithObiWarp(vector<mzSample*> samples,  ObiParams* obiParams, int referenceSampleIndex) {
```

Comment:
Just as a reminder, we decided that our opening braces for functions, methods and namespaces will be on the next line. Space-separation for all other opening braces.